### DISCUSSION TITLE
Encja daty ostatniej aktualizacji Taurona

### DISCUSSION THREAD

**Discussion by robertfili** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#148] [Q&A] [answered]
Nie mogę sobie poradzić, żeby dodać encję do Taurona, która pokazywałaby datę, ewentualnie również godzinę ostatniej aktualizacji danych. Mój stopień zaawansowania to 0+
Z góry dziękuję za podpowiedź

**Comment by PiotrMachowski** [comment, marked as answer]
Możesz użyć takiego szablonu w [template sensorze](https://www.home-assistant.io/integrations/template/#state-based-template-binary-sensors-buttons-numbers-selects-and-sensors):

```
{{ states.sensor.tauron_amiplus_123456_daily_energy_consumption.last_changed }}
```

**Comment by robertfili** [comment]
Zgadza się 👍👍👍
